mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -124,71 +127,82 @@ function Row(props: {
-                                'removeJoint',
-                              );
-                            }}
-                          >
-                            Delete
-                          </Sbutton>
-                        </>
-                      )}
-                    </div>
+                  <br/>
+                </div>
+                <div className="col-5 mt-1 mx-3">
+                  {props.row.notes &&
+                      <>
+                          <div>Notes:</div>
+                          <div className="fw-normal mt-2">{props.row.notes}</div>
+                      </>
+                  }
+                </div>
+                <div className='col mt-4 d-flex flex-row gx-0 justify-content-end'>
+                  <div className="d-flex flex-column action-buttons">
+                    {props.isEditModeActive && (
+                      <>
+                        <Sbutton
+                          onClick={(e) => {
+                            props.callBackFunction.performAccordionAction(
+                              props.row,
+                              'openMappingDetails',
+                            );
+                          }}
+                        >
+                          Edit
+                        </Sbutton>
+                        <Sbutton
+                          className="mt-2"
+                          onClick={(e) => {
+                            props.callBackFunction.performAccordionAction(
+                              props.row,
+                              'removeMapping',
+                            );
+                          }}
+                        >
+                          Delete
+                        </Sbutton>
+                      </>
+                    )}
                   </div>
                 </div>
               </div>
+            </div>
           </Collapse>
         </TableCell>
       </StyledTableRow>
     </>
   );
 }
 
-export default function JointListingAccordion(props: any) {
+function filterMappings(nodeMappingsInput: NodeMapping[], value: string, showAttributeNames: boolean) {
+  return nodeMappingsInput.filter(item => {
+    const searchString = value.toLowerCase();
+    return (item.source[0].label.toLowerCase().includes(searchString) || item.target[0].label.toLowerCase().includes(searchString) || (item?.notes && item.notes.toLowerCase().includes(searchString)));
+  });
+}
+
+export default function MappingsAccordion(props: any) {
+  const {t} = useTranslation('common');
+  const [mappingData, setMappingData] = React.useState<NodeMapping[]>([]);
+  const [showAttributeNames, setShowAttributeNames] = React.useState<boolean>(true);
   useEffect(() => {
-//console.log('props changed', props);
+    setMappingData(props.nodeMappings);
+    setShowAttributeNames(props.showAttributeNames);
   }, [props]);
-  const nodeMappingsInput = props.nodeMappings;
+  let nodeMappingsInput = props.nodeMappings;
```

Comment:
No reassignment was added, so no need to change from const to let, just makes the linter unhappy